plugins/catalog-common/src/index.ts

```diff
@@ -30,6 +30,7 @@ export {
   catalogLocationReadPermission,
   catalogLocationCreatePermission,
   catalogLocationDeletePermission,
+  catalogPermissions,
```

Comment:
How do we feel about exporting the list alongside the individual permissions like this? If we want to consolidate the export into a single object with all the permissions as fields, I can certainly do that - that PR will be much bigger though.

Reply:
Seems reasonable to me! We have precedent for doing the same thing for [rules](https://github.com/backstage/backstage/blob/master/plugins/catalog-backend/src/permissions/rules/index.ts#L30). I wonder if we should use an object rather than array here too? That way other consumers can use this as a convenient way to import all permissions and access specific ones, whereas with an array they'll need to walk the list to find the right one.

Reply:
My rationale with the array was that if you want the individual permissions you can just grab them directly, or if you want the whole thing, you'd most likely want it as a list (like in our case). Though if we are rather taking the approach of "hey, from now on if you want to import the permissions you should grab this new object and use its field", then I can definitely go ahead and change it!